### ISSUE TITLE
[Request] keep niagara media widget even if not playing anything

### ISSUE THREAD

**Issue by JONI350** [8bitPit/Niagara-Issues#1787]
### Is your feature request related to a problem?

A problem, i, usually uses youtube music for listening things and then when i close all things my musics stops and i just hit play to continue, similarly i use niagara widget for when i hear podcasts on spotfy, however as i cant fix niagara media widget it always get messy and its making me have issues with it, opposite of being a utility basically

![Screenshot_20211227-194556_Niagara Launcher](https://user-images.githubusercontent.com/25126715/147512042-56d80e03-0f50-4230-a543-1b18bbbb18b3.jpg)

Here's how i like to use it, problem being that i just cant close the app and use only the widget as i can with youtube music

### Describe the solution you'd like

Basically a toggle "keep media widget on the screen"

**Comment by 8bitPit** [comment]
Hi @JONI350, I'm sorry but we cannot support that, see: https://help.niagaralauncher.app/article/11-media-widget#:~:text=Why%20doesn%27t%20the,last%20media%20session).
